packages/nextjs/services/onchainImpactDashboardApi/globalScoreServices.ts

```diff
@@ -0,0 +1,13 @@
+import { GlobalScoreDTO } from "~~/pages/api/stub/globalScore";
```

Comment:
GlobalScore shouldn't be in the stubs directory. Perhaps we should rename the "stubs" directory to "externalStubs" so that it is clear it is only for stubbing out the apis that are external to us. What do you think?